client/src/test/omtLinkProvider.test.ts

```diff
@@ -1,60 +1,31 @@
-import { DocumentLink, commands, workspace, Uri, Position, Range } from 'vscode';
+import { DocumentLink, Position, Range } from 'vscode';
 import { getDocUri, activate } from './helper';
-import { expect } from 'chai';
+import { SinonStub, stub, assert } from 'sinon';
+import { LanguageClient } from 'vscode-languageclient';
 
 describe('OMTLinkProvider', () => {
     const docUri = getDocUri('one/imports.omt');
-    const expectedCount = 6;
-    let actualDocumentLinks: DocumentLink[];
+    let sendRequestStub: SinonStub;
 
     before(async () => {
+        sendRequestStub = stub(LanguageClient.prototype, 'sendRequest')
```

Comment:
`stub` need to be `.restore()` or `.reset()`

Reply:
yes, it just did not cause any issues because it was used only once. 
so, done